swyft/store/simulator.py

```diff
@@ -128,6 +131,12 @@ def from_command(
                 Each instance of the simulator will run in a separate
                 sub-folder. It must exist.
         """
+        if shell:
+            log.warning(
+                "Your command-line program will run through a shell - check the "
+                "following security considerations: "
+                "https://docs.python.org/3/library/subprocess.html#security-considerations"
+            )
```

Comment:
@bkmi: as agreed we also warn the user about the security risk with `shell=True`